tensorboard/webapp/widgets/card_fob/card_fob_controller_component.ts

```diff
@@ -379,6 +379,32 @@ export class CardFobControllerComponent {
     });
   }
 
+  prospectiveFobClicked(event: Event) {
+    if (!this.isProspectiveFobFeatureEnabled) {
+      return;
+    }
+    event.stopPropagation();
+    if (this.prospectiveStep === null) {
```

Comment:
Again it seems like that ngIf should already prove this. Is this just for typescripts sake?

Reply:
Yeah, this is mostly for TypeScript but this COULD technically happen because the clickable area is still shown if the `prospectiveStep` is null.

Reply:
Hmm. Is there a reason for that? Out of the scope of this PR...just curious.

Reply:
Yeah, the mouse hovering clickable area is how the prospective step gets set.